app/src/main/java/woowacourse/omok/MainActivity.kt

```diff
@@ -0,0 +1,141 @@
+    }
+
+    private fun setOnClickListener(view: ImageView, index: Int, turn: State) {
+        view.setOnClickListener {
+            when (turn as Turn) {
+                Turn.Black -> blackTurn(index, view)
+                Turn.White -> whiteTurn(index, view)
+            }
+        }
+    }
+
+    private fun blackTurn(index: Int, view: ImageView) {
+        val position = positionFind(index)
+        if (!omokBoard.isBlackPlaceable(position)) {
+            Toast.makeText(applicationContext, BOARD_PUT_ERROR_MESSAGE, Toast.LENGTH_SHORT).show()
+            turn(Turn.Black)
+        }
+        if (omokBoard.isBlackPlaceable(position)) {
+            dBController.insertDB(Turn.Black.color, index)
+            view.setImageResource(R.drawable.black_stone)
+            omokBoard.blackPlayer.put(BlackStone(position))
+            omokBoard.occupyPosition(position)
+            nextStepBlack(position)
+        }
+    }
+
+    private fun nextStepBlack(position: Position) {
+        when (lineJudge(omokBoard.blackPlayer, BlackStone(position))) {
+            true -> turn(Win.Black)
+            false -> turn(Turn.White)
+        }
+    }
+
+    private fun whiteTurn(index: Int, view: ImageView) {
+        val position = positionFind(index)
+        if (!omokBoard.isWhitePlaceable(position)) {
+            Toast.makeText(applicationContext, BOARD_PUT_ERROR_MESSAGE, Toast.LENGTH_SHORT).show()
+            turn(Turn.White)
+        }
+        if (omokBoard.isWhitePlaceable(position)) {
+            dBController.insertDB(Turn.White.color, index)
+            view.setImageResource(R.drawable.white_stone)
+            omokBoard.whitePlayer.put(WhiteStone(position))
+            omokBoard.occupyPosition(position)
+            nextStepWhite(position)
+        }
+    }
+
+    private fun nextStepWhite(position: Position) {
+        when (lineJudge(omokBoard.whitePlayer, WhiteStone(position))) {
+            true -> turn(Win.White)
+            false -> turn(Turn.Black)
+        }
+    }
+
+    fun positionFind(index: Int): Position {
+        val x = (index % Position.MAX_VERTICAL_AXIS) + POSITION_CALCULATE_PLUS_NUMBER
+        val y = Position.MAX_VERTICAL_AXIS - index / Position.MAX_VERTICAL_AXIS
+        return Position(HorizontalAxis.getHorizontalAxis(x), y)
+    }
+
+    private fun lineJudge(player: Player, stone: Stone) = LineJudgement(player, stone).check()
+
+    private fun goWinActivity(state: Win) {
+        val color = when (state) {
+            Win.Black -> Turn.Black.color
+            Win.White -> Turn.White.color
+        }
+        val intent = Intent(this, WinActivity::class.java)
+        intent.putExtra(OmokConstract.TABLE_COLUMN_COLOR, color)
+        startActivity(intent)
+    }
+
+    override fun onDestroy() {
+        dBController.closeDB()
+        super.onDestroy()
+    }
+
+    companion object {
+        const val POSITION_CALCULATE_PLUS_NUMBER = 1
+        const val BOARD_PUT_ERROR_MESSAGE = "해당 자리에 둘 수 없습니다."
```

Comment:
```suggestion
        private const val POSITION_CALCULATE_PLUS_NUMBER = 1
        private const val BOARD_PUT_ERROR_MESSAGE = "해당 자리에 둘 수 없습니다."
```
내부에서 사용하는 변수라면, private 접근자를 붙여보면 어떨까요?

Reply:
넵! private 접근자로 변경했습니다!